csharp/ForgettingTests.cs

```diff
@@ -2,19 +2,6 @@
 /// test needs to be removed when you remove the feature flag.
 public class ForgettingTests
 {
-    /// This is an existing test built before your feature was introduced
-    [Test]
-    // FIX: [FeatureFlag("flag-name", false)]
-    public void Method_NormalOperation_ReturnsTrue()
```

Comment:
You'll only find out that this test needs removing AFTER you build + run tests.
It might be a bit disruptive and you'll need to spend time verifying the results.